src/components/KRadio/KRadio.vue

```diff
@@ -153,155 +173,233 @@ const modifiedAttrs = computed(() => {
 </script>
 
 <style lang="scss" scoped>
-@import '@/styles/mixins';
+/* Component variables */
+
+$kRadioDotSize: 6px;
+$kRadioDotBackground: var(--kui-color-background, $kui-color-background);
+$kRadioBackground: var(--kui-color-background, $kui-color-background);
+$kRadioBorder: var(--kui-shadow-border, $kui-shadow-border);
+$kRadioHoverBorder: var(--kui-shadow-border-primary-weak, $kui-shadow-border-primary-weak);
+// focus
+$kRadioFocusRing: var(--kui-shadow-focus, $kui-shadow-focus);
+// active
+$kRadioActiveBackground: var(--kui-color-background-primary-strongest, $kui-color-background-primary-strongest);
+$kRadioActiveBorder: var(--kui-shadow-border-primary-strongest, $kui-shadow-border-primary-strongest);
+// checked
+$kRadioCheckedBackground: var(--kui-color-background-primary, $kui-color-background-primary);
+$kRadioCheckedBorder: var(--kui-shadow-border-primary, $kui-shadow-border-primary);
+// error
+$kRadioErrorBackground: var(--kui-color-background-danger, $kui-color-background-danger);
+$kRadioErrorBorder: var(--kui-shadow-border-danger, $kui-shadow-border-danger);
+$kRadioErrorHoverBorder: var(--kui-shadow-border-danger-strong, $kui-shadow-border-danger-strong);
+// disabled
+$kRadioDisabledBackground: var(--kui-color-background-disabled, $kui-color-background-disabled);
+$kRadioDisabledBorder: var(--kui-shadow-border-disabled, $kui-shadow-border-disabled);
+$kRadioDisabledDotBackground: var(--kui-color-background-neutral-weak, $kui-color-background-neutral-weak);
```

Comment:
🔴 **issue**: we definitely do not want to map all of the CSS custom properties to local variables; they should be used inline. 

I didn't notice this pattern in other components, but if this exists, please revert and utilize the CSS custom properties inline as expected.